homeassistant/components/roborock/__init__.py

```diff
@@ -28,6 +28,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+type RockorockConfigEntry = ConfigEntry[RoborockCoordinators]
```

Comment:
Should this be `Roborock`?
```suggestion
type RoborockConfigEntry = ConfigEntry[RoborockCoordinators]
```

Reply:
Ups my dog names is Rocko :sweat_smile: